packages/esm-patient-medications-app/src/order-basket/order-basket-search/order-basket-search-results.component.tsx

```diff
@@ -132,43 +132,26 @@ const DrugSearchResultItem: React.FC<DrugSearchResultItemProps> = ({ drug, onSea
 
 const Skeleton = () => {
   const isTablet = useLayoutType() === 'tablet';
+  const tileClassName = `${isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`} ${
+    styles.skeletonTile
+  }`;
   return (
     <div className={styles.searchResultSkeletonWrapper}>
       <div className={styles.orderBasketSearchResultsHeader}>
         <SkeletonText className={styles.searchResultCntSkeleton} />
         <ButtonSkeleton size={isTablet ? 'md' : 'sm'} />
       </div>
-      <Tile
-        className={`${isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`} ${
-          styles.skeletonTile
-        }`}
-        onClick={() => {}}
-      >
+      <Tile className={tileClassName} onClick={() => {}}>
```

Comment:
Little thing - just create a function outside of `Skeleton` like this:

```js
function noop() {}
```

then you can always use `onClick={noop}`. No need to create new references etc.

Reply:
I removed the onClick. Thanks for the catch!